test-result-summary-client/src/Dashboard/Widgets/DateRangePickers.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
-import moment from 'moment';
 import DayPickerInput from 'react-day-picker/DayPickerInput';
+import dateFnsFormat from 'date-fns/format';

```

Comment:
I have the same concern as Lan. We should stick to `moment` for dealing with dates since there's no need for using another library.